torch/special/__init__.py

```diff
@@ -230,6 +230,48 @@
     tensor([ 0.,  1.])
 """.format(**common_args))
 
+xlog1py = _add_docstr(_special.special_xlog1py,
+                      r"""
+xlog1py(input, other, *, out=None) -> Tensor
+
+Computes ``input * log1p(other)`` with the following cases.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        \text{NaN} & \text{if } \text{other}_{i} = \text{NaN} \\
```

Comment:
I have one nit: what if other_i is nan and input_i is zero?

Maybe the second case needs a "if input_i is zero and other_i is not NaN"?

Reply:
Right. That makes it much clearer.